deploy-airflow-dags/action.yml

```diff
@@ -0,0 +1,49 @@
+name: "Deploy Airflow dags"
+description: Deploy dags to Airflow instance
+branding:
+  icon: refresh-ccw
+  color: green
+inputs:
+  project:
+    required: true
+    description: 'Name of the project'
+  version:
+    required: true
+    description: 'Release version of the project'
+  airflow_api_url:
+    required: true
+    description: 'Airflow instance Api url'
+  airflow_username:
+    required: true
+    description: 'Airflow instance username'
+  airflow_password:
+    required: true
+    description: 'Airflow instance password'
+  airflow_dags_path:
```

Comment:
suggestion: this is the path of the dags folder relative to the project this action is launched on, maybe a more explanatory names helps who use it to understand it better, also maybe we should introduce a check that if that dir is not present the action fails